internal/cmd/deployrequest/dr.go

```diff
@@ -20,6 +20,9 @@ func DeployRequestCmd(ch *cmdutil.Helper) *cobra.Command {
 	cmd.PersistentFlags().StringVar(&ch.Config.Organization, "org", ch.Config.Organization, "The organization for the current user")
 	cmd.MarkPersistentFlagRequired("org") // nolint:errcheck
 
+	cmd.AddCommand(ApplyCmd(ch))
+	cmd.AddCommand(EditCmd(ch))
+	cmd.AddCommand(CancelCmd(ch))
```

Comment:
The list is sorted alphabetically, so it would be great if you could move `EditCmd` into the appropriate place.